web/build.gradle.kts

```diff
@@ -52,6 +52,7 @@ tasks {
   }
 
   clean {
+    delete(".gradle")
```

Comment:
is this necessary to delete .gradle?

Reply:
Yes, to prevent issues where users had downloaded an older version causing incompatibility with the current pnpm-lock.yaml.

Reply:
After we fix the pnpm version in gradle, do we still need to clean this folder?

Reply:
If the pnpm version in web/.gradle is 8 or earlier, and running build won't update this folder, so this issue will still occur.
Either clean delete this folder or remove it manually.

Reply:
The previous concern was to try to avoid problems as much as possible by deleting .gradle.
After testing several different possibilities, it turns out there is no issue, so we don't need to delete .gradle.
I will revert it now.

Reply:
IMO, we can delete the .gradle folder which is a ignore folder. 
Delete the .gradle every time build will be consistent with the remote, similar upgrade issues can be found in local builds.

Deleting it will definitely not reproduce the issue of update pnpm-lock.yaml, but not deleting it will require retesting to make sure it doesn't.